packages/create-react-native-library/templates/example/example/ios/Podfile

```diff
@@ -16,8 +16,17 @@ target '<%- project.name %>Example' do
   #
   # Note that if you have use_frameworks! enabled, Flipper will not work and
   # you should disable these next few lines.
-  use_flipper!({ 'Flipper' => '0.80.0' })
+  # flipper does not work with detox 
+  # use_flipper!({ 'Flipper' => '0.80.0' })
+  # post_install do |installer|
+  #   flipper_post_install(installer)
+  # end
```

Comment:
Is there a way to conditionally check if we're building release build/on CI and exclude Flipper? Maybe some env variable? There's `process.env.CI` for CI.

I'd like to keep Flipper since it's essential for debugging stuff like Reanimated since it's not possible to use Chrome debugger, and would be important as more and more modules start using JSI.

Reply:
Good point, I will try to fix it.